plugin.video.skyvideoitalia/resources/settings.xml

```diff
@@ -1,7 +1,9 @@
 <?xml version="1.0" encoding="utf-8" standalone="yes"?>
 <settings>
     <category label="30000">
-        <setting id="Quality" type="enum" label="30002" default="3" values="LOW|MED|HIGH|HD" />
-        <setting id="LogLevel" type="enum" label="30003" default="3" values="Debug|Info|Warning|Error"/>
+        <setting label="30002" id="Quality" type="enum" default="3" values="LOW|MED|HIGH|HD" />
```

Comment:
would be nice if at some point you could update the settings to the new format: https://forum.kodi.tv/showthread.php?tid=350135